src/ContentRepository90/Rules/FusionNodeHiddenInIndexRector.php

```diff
@@ -26,17 +26,14 @@ public function refactorFileContent(string $fileContent): string
                 $eelExpression
             ))
             ->addCommentsIfRegexMatches(
-                '/\.hiddenInIndex/',
+                '/\.hiddenInIndex$/',
```

Comment:
so it will not warn about `fooo.hiddenInIndex ? true : false` anymore because its not at the end? `$`?

Reply:
True. 😞

Reply:
sorry